README.md

```diff
@@ -156,7 +156,7 @@ java -Dlogback.configurationFile=/path/to/logback-custom.xml -jar eclair-node-gu
   receive      | amountMsat, description                                                                | generate a payment request for a given amount
   receive      | amountMsat, description, expirySeconds                                                 | generate a payment request for a given amount that expires after given number of seconds
   checkinvoice | paymentRequest                                                                         | returns node, amount and payment hash in an invoice/paymentRequest
-  findroute    | paymentRequest|paymentRequest, amountMsat|nodeId, amountMsat                           | given a payment request or nodeID and an amount checks if there is a valid payment route returns JSON with attempts, nodes and channels of route
+  findroute    | [paymentRequest], [paymentRequest, amountMsat], [nodeId, amountMsat]                   | returns nodes and channels of the route if there is any
```

Comment:
This not consistent with the current format, it should be one line per set of arguments:

```
findroute    | paymentRequest, amountMsat = ?                             | returns nodes and channels of the route if there is any
findroute    | nodeId, amountMsat                                         | returns nodes and channels of the route if there is any
```